db/seeds.rb

```diff
@@ -5,3 +5,108 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+# Default admin
+
+User.create(
+  email: "admin@subway-reviews.com",
+  password: "password",
+  password_confirmation: "password",
+  role: "admin"
+  )
+
+lines = Line.create([
+  {name: "Red"},
+  {name: "Green"},
+  {name: "Blue"},
+  {name: "Orange"},
+  ])
+
+require 'open-uri'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.